setup.py

```diff
@@ -19,54 +19,54 @@ def read(fname):
     "requests",
     "urllib3>=1.26.0",
 ]
-tests_require = ['pytest']
+tests_require = ["pytest"]
 
 # We can't get the values using `from pyfcm import __meta__`, because this would import
 # the other modules too and raise an exception (dependencies are not installed at this point yet).
 meta = {}
-exec(read('pyfcm/__meta__.py'), meta)
+exec(read("pyfcm/__meta__.py"), meta)
 
-if sys.argv[-1] == 'publish':
+if sys.argv[-1] == "publish":
     os.system("rm dist/*.gz dist/*.whl")
-    os.system("git tag -a %s -m 'v%s'" % (meta['__version__'], meta['__version__']))
+    os.system("git tag -a %s -m 'v%s'" % (meta["__version__"], meta["__version__"]))
     os.system("python setup.py sdist bdist_wheel")
     os.system("twine upload dist/*")
     os.system("git push --tags")
     sys.exit()
 
 setup(
-    name=meta['__title__'],
-    version=meta['__version__'],
-    url=meta['__url__'],
-    license=meta['__license__'],
-    author=meta['__author__'],
-    author_email=meta['__email__'],
-    description=meta['__summary__'],
-    long_description=read('README.rst'),
-    packages=['pyfcm'],
+    name=meta["__title__"],
+    version=meta["__version__"],
+    url=meta["__url__"],
+    license=meta["__license__"],
+    author=meta["__author__"],
+    author_email=meta["__email__"],
+    description=meta["__summary__"],
+    long_description=read("README.rst"),
+    packages=["pyfcm"],
```

Comment:
```suggestion
    packages=["pyfcm", "pyfcm.v1"],
```
`pyfcm.v1` is missing

Reply:
Hi @wkpark ,
Good catch.

Reply:
Hi @olucurious ,
Do we need this to be mentioned in the packages list?